### DISCUSSION TITLE
Falha na validação do campo AuditFile.Header.ProductID

### DISCUSSION THREAD

**Discussion by 3tipo** [assoft-portugal/SAF-T-AO#187] [General]
Pessoal Podem me Ajudar nestes erros:
Erro no ficheiro, AuditFile.MasterFiles.1Customer.CustomerTaxID é inválido.
Falha na validação do campo AuditFile.Header.ProductID
Nome da Empresa:
PEDRO MBOYO - PRESTAÇAO DE SERRVIÇOS E COMERCIO GERAL, LDA
NIF do Produtor de Software: 5000497509
i
* Nome do Programa: FaturVenda
i
* Versão: 1.0.0
i
Marca : Soffácil
**Qual deve ser o nome do ProductId? e Porque está a dar erro no AuditFile.MasterFiles.1Customer.CustomerTaxID?**

**Comment by 3tipo** [comment]
![imagemteste](https://user-images.githubusercontent.com/71136363/131654988-55bba45a-0da0-4106-b18b-051d6aeafa25.JPG)

**Reply by angelobarata** [reply]
Também tenho a mesma dificuldade, alguém aí já ultrapassou?
